gallery/src/GalleryNav/GalleryNav.scss

```diff
@@ -0,0 +1,13 @@
+.gallery-nav-link {
+  color: var(--nx-color-text-dark);
+
+  .fa-sass {
+    // Optical horizontal alignment adjustment
+    margin-left: calc(var(--nx-spacing-2x) + 0.08em);
```

Comment:
This seems suspicious.  It makes the text for those rows unaligned with the others and otherwise seems unneeded as far as I can perceive.

Reply:
> This seems suspicious. It makes the text for those rows unaligned with the others and otherwise seems unneeded as far as I can perceive.

https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/4be88b832d879c22b01aa0d55ad92b27e75efb20